streaming/src/main/scala/org/apache/spark/streaming/testutil/StreamingTestUtils.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.streaming.testutil
+
+import org.apache.spark.SparkContext
+import org.apache.spark.internal.Logging
+import org.apache.spark.streaming.StreamingContext
+
+object StreamingTestUtils extends Logging {
```

Comment:
No big deal but is it useful to just have these utility methods in `LocalStreamingContext`, in a companion object?

Reply:
Yeah that sounds better. Actually StreamingTestUtils was introduced to apply the change across different modules - I thought we don't prefer to depend on test jar, but just realized it's already widely used so the original purpose is invalid right now.